### ISSUE TITLE
Do we have any plan to add filtering criteria to `all_collections` or add another function that can list filtered collections? 

### ISSUE THREAD

**Issue by Geosynopsis** [stac-utils/pgstac#139]
Use-Case: Restricting collections based on ACL

**Comment by bitner** [comment]
There is no specification in STAC as far as I know for filtering calls to /collections.

You can search items specifying only specific collections using /search?collections=collectiona,collectionb

**Comment by Geosynopsis** [comment]
Yes, I understand from the endpoint's point of view. But is it also translatable to the database functions?

**Comment by bitner** [comment]
I do not have any plans for adding this, but i am open to pull requests.

**Comment by bitner** [comment]
I don't think that we'll be addressing this directly in pgstac anytime soon, but between collection search that has been added since this ticket was opened and stac-auth-proxy, I think you should be able to get what you want.
